spec/lexical-structure.md

```diff
@@ -110,7 +110,7 @@ new_line_character
     ;
 
 delimited_comment
-    : '/*' delimited_comment_section* asterisk* '/'
+    : '/*' delimited_comment_section* asterisk* '*/'
```

Comment:
The correct fix is to change `asterisk*` to `asterisk+`.

A similar change needs to be made in `delimited_comment_section` to remove an ambiguity.

Reply:
@gafter I'll change `asterisk*` to `asterisk+`; just want to check with you that suggested fix is also correct. I've chosen that variant to match the sentence:
>**Delimited comments** start with the characters `/*` and end with the characters `*/`.

I don't understand your remark about `delimited_comment_section`. Similar change means
```antlr
delimited_comment_section
    : '/'
    | asterisk+ not_slash_or_asterisk
    ;
```

Reply:
I was wrong about `delimited_comment_section`.